smarts/core/scenario.py

```diff
@@ -44,6 +44,7 @@
 from .utils.file import file_md5_hash, make_dir_in_smarts_log_dir, path2hash
 from .utils.id import SocialAgentId
 from .utils.math import vec_to_radians
+from .utils.traffic_history_serivce import Traffic_history_service
 from .waypoints import Waypoints
```

Comment:
Always use full paths rather than relative paths.

Reply:
It seems many places in the code base uses relative paths, for example `smarts.py`. Converting 1 relative path to full path doesn't affect much, we should either convert all of them or not convert.

Reply:
Indeed, I was suggesting to change all import paths to use full paths whenever you encounter them. I would prefer to always use full import paths compared to relative import paths. I do recognise the previous authors used relative paths, but I believe this is not a good practise. It leads to confusion and prone to error when the files are moved around.